archive/p/perl6/README.md

```diff
@@ -0,0 +1,9 @@
+# Sample Programs in Perl 6
+
+Welcome to Sample Programs in Perl 6!
+
+## Sample Programs
+
+- [Hello World in Perl 6][1]
+
```

Comment:
Looks like there are a couple typical sections missing:

- Fun Facts
- References

Any chance you could include them?